downloaded/installLisk.sh

```diff
@@ -54,6 +54,19 @@ prereq_checks() {
 		echo "Error: tar is not installed. Exiting."
 		exit 2
 	fi
+
+	if command -v ss &>/dev/null ; then
+		PORT_5432_IN_USE=$(ss -tna | grep -c 'LISTEN.*:5432 ')
```

Comment:
I think this shoud read `ss -tnl` (or better yet: `ss --tcp --numeric --listening`)